.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I suggest we leave lint-staged

with previous configuration it did not commit the lint --fix, as there was no 'git add' part of lint-staged that was previously running with husky4.x

Reply:
I don't see the interest to duplicate commandes. `lint-staged`, `lint`and `lint-chacked`are almost the same.

Like I said in another comment, I always forget to run lint-check. So it's always fails in the CI. And it's very frustrating and tiring having to go back over your code when you thought you were done.